r11701/running_scripts/parameter_space_v2/create_csv.ipynb

```diff
@@ -92,14 +93,35 @@
     "Z_max = 10**0\n",
     "Z_n = 5\n",
     "met = log_range(Z_min,Z_max,Z_n)*Zsun\n",
-    "# metallicity extentions\n",
-    "met = np.array([2.,0.2])*Zsun\n",
-    "met = np.array([0.45])*Zsun # changed %0.e_Zsun in favour of %0.1e_Zsun\n",
+    "met = np.concatenate([met, [0.2*Zsun, 0.45*Zsun, 2*Zsun]])\n",
```

Comment:
I'd suggest to update the Z_n, to get the printed numbers of runs correctly. (Same applies to other grids.)

Reply:
The printed number of runs is not correct anyway. Due to how `[::2]` selects systems, this is different than `#_n/2`. 
Feel free to change them to make them consistent/correct